pkgs/development/python-modules/pythonparser/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python
+, pythonAtLeast
+, regex
+}:
+
+buildPythonPackage rec {
+  pname = "pythonparser";
+  version = "1.3";
+
+  disabled = pythonAtLeast "3.7";
```

Comment:
According to https://github.com/m-labs/pythonparser/issues/20, there's not even support for Python 3.6.

Reply:
Good find. Asking for clarification.

Reply:
On the other hand, as it builds and tests on Python 3.6, it still has value there, even though it might only be able to parse 3.5-compatible source. If someone tries to use it on sourcecode it doesn't understand, they will get errors, but if they don't, it can be installed together with some hypothetical 3.6-only package they might need.